### PR TITLE
new version

### PR DESCRIPTION
**Pull request by MarieStLaurent**

### CODE REVIEW

File: Subjet_BehavData_v5.py [hunk 1]
```diff
@@ -1 +1,312 @@
+import numpy as np
+import pandas as pd
+import os, re
+import glob
```
Comment (arnaudbore) [inline Subjet_BehavData_v5.py]: Now, you need to use glob to get all the files.
Something like this:
allMainFiles = glob.glob('Output-Responses-Encoding_CIMAQ*txt')
id = allMainFiles[0].split('_')[3]

File: Subjet_BehavData_v5.py [hunk 2]
```diff
@@ -1 +1,312 @@
...
+mainFile[['InScan_RT']]=mainFile[['InScan_RT']].astype('float64', copy=False)
+mainFile['InScan_RT'] = mainFile['InScan_RT'].div(1000)
+
+#Remove Col8: 'Stim_ACC' (redundant)
+mainFile.drop('Stim_ACC', axis=1, inplace=True) #axis 0 = rows, 1 = columns
+
+#Load other enconding file with onset times (encOnsets): Onset-Event-Encoding_CIMAQ_*.txt
+##Data is Fixed Width, padding with blank spaces
+#Column format is estimated from first 100 rows by default; reset to avoid truncated values
+encOnsets = pd.read_fwf('Onset-Event-Encoding_CIMAQ_'+str(bID)+'*.txt', infer_nrows=210, delim_whitespace=True, header=None)
```
Comment (arnaudbore) [inline Subjet_BehavData_v5.py]: Dummy question: Is this cautious to hard code infer_nrows ?

File: Subjet_BehavData_v5.py [hunk 3]
```diff
@@ -1 +1,312 @@
...
+#PostScan_SourceRT
+#PostScan_SourceACC (to be calculated by cross-referencing encoding and retrieval output files)
+mainFile.insert(loc=mainFile.shape[1], column='StimCategory', value='tbd', allow_duplicates=True)
+mainFile.insert(loc=mainFile.shape[1], column='StimulusImage', value='tbd', allow_duplicates=True)
+mainFile.insert(loc=mainFile.shape[1], column='PostScanReco_ACC', value=-1, allow_duplicates=True)
+mainFile.insert(loc=mainFile.shape[1], column='PostScanReco_RT', value=-1.0, allow_duplicates=True)
+mainFile.insert(loc=mainFile.shape[1], column='PostScan_SourceResp', value=-1, allow_duplicates=True)
+mainFile.insert(loc=mainFile.shape[1], column='PostScan_SourceRT', value=-1.0, allow_duplicates=True)
+mainFile.insert(loc=mainFile.shape[1], column='PostScan_SourceACC', value=-1, allow_duplicates=True)
+
```
Comment (arnaudbore) [inline Subjet_BehavData_v5.py]: We could reduce these lines :

> for n in ['StimCategory','StimulusImage','PostScanReco_ACC','PostScanReco_RT','PostScan_SourceResp','PostScan_SourceRT','PostScan_SourceAC']:
> mainFile.insert(loc=mainFile.shape[1], column=n, value='tbd', allow_duplicates=True)

File: Subjet_BehavData_v5.py [hunk 4]
```diff
@@ -1 +1,312 @@
...
+        if retriev.loc[i, 'Recognition_ACC'] == 1:
+            retriev.loc[i, 'Hit']=1
+        else:
+            retriev.loc[i, 'Miss']=1
+    else:
+        if retriev.loc[i, 'Recognition_ACC'] == 1:
+            retriev.loc[i, 'CorrectRej']=1
+        else:
+            retriev.loc[i, 'FAlarm']=1
+
```
Comment (arnaudbore) [inline Subjet_BehavData_v5.py]: There's must be a way to that without the loop.

File: Subjet_BehavData_v5.py (hunk 2 above)
Comment (MarieStLaurent) [reply Subjet_BehavData_v5.py]: If I don't, the default is 100, and it leads to some of my data being truncated in rows 200s+ (the trial numbers are missing their last digit)

File: subject_behavdata.py [hunk 5]
```diff
@@ -0,0 +1,269 @@
+
```
Comment (arnaudbore) [inline subject_behavdata.py]: Add

```
#!/usr/bin/env python
# encoding: utf-8
```

File: subject_behavdata_manual.py [hunk 6]
```diff
@@ -0,0 +1,268 @@
+
```
Comment (arnaudbore) [inline subject_behavdata_manual.py]: Add

```
#!/usr/bin/env python
# encoding: utf-8
```

**Pull request merged**